src/celldega/nbhd/__init__.py

```diff
@@ -205,3 +211,108 @@ def alpha_shape_geojson(gdf_alpha, meta_cluster, inst_alpha):
     geojson_alpha['inst_alpha'] = inst_alpha
 
     return geojson_alpha
+
+def create_hextile(radius, path_landscape_files=None, img_height=100, img_width=100, pixel_size=0.2125):
+
+    if isinstance(path_landscape_files, str):
```

Comment:
This is ok for now, but we should add the image height/width to the `landscape_parameters.json` in a future pull request (DEGA-244).